lib/protobuf/field/message_field.rb

```diff
@@ -9,7 +9,7 @@ class MessageField < BaseField
       #
 
       def acceptable?(val)
-        val.is_a?(type_class) || val.respond_to?(:to_hash)
+        val.is_a?(type_class) || val.respond_to?(:to_hash) || val.respond_to?(:to_proto)
```

Comment:
Definitely in favor of this!

![bacon](https://cloud.githubusercontent.com/assets/207933/13544625/781e9bba-e235-11e5-89bb-47728fb5e88e.gif)